tests/Bridge/Symfony/Doctrine/PhpcrLoaderIntegrationTest.php

```diff
@@ -46,6 +46,7 @@ public static function setUpBeforeClass(): void
 
     public function setUp(): void
     {
+        $this->markTestSkipped('annotation_reader is not available');
```

Comment:
Just to avoid to have to come back to the PR when checking this comment:

```suggestion
  if (PHP_VERSION_ID < 80000) {
        $this->markTestSkipped('The annotation reader is not available: the "enable_annotations" on the validator cannot be set as the PHP version is lower than 8');
  }
```